main.tf

```diff
@@ -12,8 +12,51 @@ data "aws_caller_identity" "current" {}
 
 data "aws_region" "current" {}
 
+data "http" "network" {
+  url = "https://s3-eu-west-1.amazonaws.com/monitoring-jump-start/data/network.json"
```

Comment:
Why not include the `network.json` in the module?

Reply:
I haven't changed this in this PR but it could be an option to bundle the file, yes. I created #2